sample/app/package.json

```diff
@@ -12,11 +12,11 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "express": "^4.16.3",
-    "handlebars": "^4.7.7",
+    "handlebars": "^4.7.6",
```

Comment:
Why did we bump this down?

Reply:
This must have been from audit fix.. I did not do it on my own